src/main/java/edu/uc/inventorymanager/dao/RetrofitClientInstance.java

```diff
@@ -0,0 +1,20 @@
+package edu.uc.inventorymanager.dao;
+
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class RetrofitClientInstance {
+
+    private static Retrofit retrofit;
+    private static String BASE_URL_ = "http://api.nobelprize.org/";
```

Comment:
Why is this link in here? We only need to either create or consume JSON so I don't think we need a client instance we can just create

Reply:
That is to consume a json for a user. I could not find one that has a random list of people so I went with nobel prized winners.